cmake/toolchain.cmake

```diff
@@ -8,6 +8,10 @@ set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
 set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)
 
+# Don't inherit compiler flags from the environment
+set(ENV{CFLAGS})
+set(ENV{CXXFLAGS})
```

Comment:
Wouldn't it be better to use [`unset()`](https://cmake.org/cmake/help/v3.6/command/unset.html) here?

Reply:
Fixed